src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -1,324 +1,53 @@
-
-            target.Run(new string[] { argName, "20" });
-            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o =>
-                o.Thresholds.Break == 20), It.IsAny<IEnumerable<LogMessage>>()));
-        }
-
-        [Theory]
-        [InlineData("--threshold-low")]
-        [InlineData("-tl")]
-        public void StrykerCLI_WithCustomThresholdLowParameter_ShouldPassThresholdLowToStryker(string argName)
-        {
-            var options = new StrykerOptions();
-            var runResults = new StrykerRunResult(options, 0.3);
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>())).Returns(runResults);
-
-            var target = new StrykerCLI(mock.Object);
-
-            target.Run(new string[] { argName, "65" });
-
-            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o =>
-                o.Thresholds.Low == 65), It.IsAny<IEnumerable<LogMessage>>()));
-        }
-
-        [Theory]
-        [InlineData("--threshold-high")]
-        [InlineData("-th")]
-        public void StrykerCLI_WithCustomThresholdHighParameter_ShouldPassThresholdHighToStryker(string argName)
-        {
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            var options = new StrykerOptions();
-            var runResult = new StrykerRunResult(options, 0.3);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>())).Returns(runResult);
-
-            var target = new StrykerCLI(mock.Object);
-
-            target.Run(new string[] { argName, "90" });
-
-            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o =>
-                o.Thresholds.High == 90), It.IsAny<IEnumerable<LogMessage>>()));
+            target = new StrykerCLI(strykerRunnerMock.Object);
         }
 
         [Fact]
-        public void StrykerCLI_OnMutationScoreBelowThresholdBreak_ShouldReturnExitCode1()
+        public void OnMutationScoreBelowThresholdBreak_ShouldReturnExitCode1()
         {
             var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            var options = new StrykerOptions(thresholdBreak: 40);
+            var options = new StrykerOptions()
+            {
+                Thresholds = new Thresholds
+                {
+                    Break = 40
+                }
+            };
             var strykerRunResult = new StrykerRunResult(options, 0.3);
 
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>())).Returns(strykerRunResult).Verifiable();
+            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerInputs>(), It.IsAny<IEnumerable<LogMessage>>())).Returns(strykerRunResult).Verifiable();
 
             var target = new StrykerCLI(mock.Object);
             var result = target.Run(new string[] { });
```

Comment:
I don't see anything asserted here (and in some following tests), am I missing something?